awx/ui_next/src/screens/InstanceGroup/Instances/InstanceListItem.jsx

```diff
@@ -56,81 +103,145 @@ function InstanceListItem({
-            <DataListCell key="type" aria-label={i18n._(t`instance type`)}>
-              <b css="margin-right: 24px">{i18n._(t`Type`)}</b>
-              <span id={labelId}>
-                {instance.managed_by_policy
-                  ? i18n._(t`Auto`)
-                  : i18n._(t`Manual`)}
-              </span>
-            </DataListCell>,
-            <DataListCell
-              key="related-field-counts"
-              aria-label={i18n._(t`instance counts`)}
-              width={2}
-            >
-              <ListGroup>
-                <b>{i18n._(t`Running jobs`)}</b>
-                <Badge isRead>{instance.jobs_running}</Badge>
-              </ListGroup>
-              <ListGroup>
-                <b>{i18n._(t`Total jobs`)}</b>
-                <Badge isRead>{instance.jobs_total}</Badge>
-              </ListGroup>
-            </DataListCell>,
-            <DataListCell
-              key="capacity"
-              aria-label={i18n._(t`instance group used capacity`)}
-            >
-              {usedCapacity(instance)}
-            </DataListCell>,
-          ]}
-        />
-        <DataListAction
-          aria-label={i18n._(t`actions`)}
-          aria-labelledby={labelId}
-          id={labelId}
-        >
-          <InstanceToggle
-            css="display: inline-flex;"
-            fetchInstances={fetchInstances}
-            instance={instance}
+          <DataListItemCells
+            dataListCells={[
+              <DataListCell key="name" aria-label={t`instance host name`}>
+                <b>{instance.hostname}</b>
+              </DataListCell>,
+              <DataListCell key="type" aria-label={t`instance type`}>
+                <b css="margin-right: 24px">{t`Type`}</b>
+                <span id={labelId}>
+                  {instance.managed_by_policy ? t`Auto` : t`Manual`}
+                </span>
+              </DataListCell>,
+              <DataListCell
+                key="related-field-counts"
+                aria-label={t`instance counts`}
+                width={3}
+              >
+                <ListGroup>
+                  <b>{t`Running jobs`}</b>
+                  <Badge isRead>{instance.jobs_running}</Badge>
+                </ListGroup>
+                <ListGroup>
+                  <b>{t`Total jobs`}</b>
+                  <Badge isRead>{instance.jobs_total}</Badge>
+                </ListGroup>
+              </DataListCell>,
+              <DataListCell
+                key="capacity-adjustment"
+                aria-label={t`capacity adjustment`}
+                width={4}
+              >
+                <SliderHolder data-cy="slider-holder">
+                  <div data-cy="cpu-capacity">{t`CPU ${instance.cpu_capacity}`}</div>
+                  <SliderForks data-cy="slider-forks">
+                    <div data-cy="number-forks">
+                      <Plural value={forks} one="# fork" other="# forks" />
+                    </div>
+                    <Slider
```

Comment:
Slider does not support ouiaId yet. @tiagodread added `data-cy` in order to ease testing. Let me know whether they are good enough.